db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
that is not a big mistake, but here is a recommendation about the naming of the related_name:
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-style
So, change it for movies